test/phoenix_live_view/integrations/html_formatter_test.exs

```diff
@@ -1,151 +1,149 @@
-if Version.match?(System.version(), ">= 1.13.0") do
```

Comment:
lol all these diffs is because if the indentation. Maybe formatter catchup on some things as well 🙌🏻